makefile.shade

```diff
@@ -38,47 +41,7 @@ use-standard-lifecycle
   exec program='${Path.Combine(SANITY_CHECKER, "bin", "Release", "SanityCheck.exe")}' commandline='${BUILD_SHARE} ${BUILD_BRANCH} ${BUILD_DIR} ${SYMBOLS_DIR} ${NUGET_SYMBOLS_SHARE} ${SYMBOLS_NUGET_EXE} '
 
 #nuget-install target='install' description='Publish all coherent packages'
-  @{
-      if (string.IsNullOrEmpty(PACKAGE_DROP_SHARE))
-      {
-          Log.Warn("PACKAGE_DROP_SHARE environment varible is not set");
-          return;
-      }
-
-      var latestPackageFolders = Directory.GetDirectories(PACKAGE_DROP_SHARE);
-      var coherentPackageNames = Directory.GetFiles(BUILD_DIR, "*.nupkg").Select(p => Path.GetFileNameWithoutExtension(p));
-
-      var delete = new List<string>();
-
-      foreach (var latestPackageFolder in latestPackageFolders)
-      {
-          var projectName = Path.GetFileName(latestPackageFolder);
-
-          var matchedVersion = new HashSet<string>(
-              coherentPackageNames
-                  .Where(packageName => packageName.IndexOf(projectName) == 0)
-                  .Select(packageName => packageName.Substring(projectName.Length + 1)));
-
-          if (matchedVersion.Any())
-          {
-              delete.AddRange(Directory.GetDirectories(latestPackageFolder)
-                                       .Where(p => !matchedVersion.Contains(Path.GetFileName(p))));
-          }
-          else
-          {
-              delete.Add(latestPackageFolder);
-              Log.Warn(string.Format("Project {0} doesn't exist in this Coherence build hence it will be deleted from {1}",
-                  projectName, PACKAGE_DROP_SHARE));
-          }
-      }
-
-      foreach (var d in delete)
-      {
-          Log.Info(string.Format("Delete folder {0}", d));
-
-          Directory.Delete(d, recursive: true);
-      }
-  }
+  exec program='${Path.Combine(FEEDS_CLEANER, "bin", "Release", "FeedsCleanup.exe")}' commandline='--coherent=${BUILD_DIR} --latest-ci=${PACKAGE_DROP_SHARE} --feed=${Environment.GetEnvironmentVariable("VOLATILE_FEED") --apikey=${Environment.GetEnvironmentVariable("APIKEY")} --whatif'
```

Comment:
Once check in the program will only be run under "whatif" mode. Once the output is verified, I will take down the `--whatif` switch.